registrations/migrations/0019_alter_registration_mandatory_fields.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.19 on 2023-08-09 11:51
+
+from django.db import migrations, models
+import registrations.models
+
+
+def migrate_mandatory_fields_field(apps, schema_editor):
+    Registration = apps.get_model("registrations", "Registration")
+
+    registrations_to_be_update = []
+    registrations = Registration.objects.filter(mandatory_fields__contains=["name"])
+
+    for registration in registrations:
+        # Replace deprecated mandatory field "name" with "first_name" and "last_name"
+        registration.mandatory_fields.append("first_name")
+        registration.mandatory_fields.append("last_name")
+        registration.mandatory_fields.remove("name")
+        registrations_to_be_update.append(registration)
+
+    Registration.objects.bulk_update(registrations_to_be_update, ["mandatory_fields"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("registrations", "0018_signup_first_and_last_name"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="registration",
+            name="mandatory_fields",
+            field=registrations.models.ChoiceArrayField(
+                base_field=models.CharField(
+                    blank=True,
+                    choices=[
+                        ("city", "City"),
+                        ("first_name", "First name"),
+                        ("last_name", "Last name"),
+                        ("phone_number", "Phone number"),
+                        ("street_address", "Street address"),
+                        ("zipcode", "ZIP code"),
+                    ],
+                    max_length=16,
+                ),
+                blank=True,
+                default=list,
+                size=None,
+                verbose_name="Mandatory fields",
+            ),
+        ),
+        migrations.RunPython(migrate_mandatory_fields_field, migrations.RunPython.noop),
```

Comment:
Since this data migration is not really backwards compatible, I would consider removing `migrations.RunPython.noop` or adding a function that reverses `migrate_mandatory_fields_field`

Reply:
Added function to reverse mandatory fields migration in https://github.com/City-of-Helsinki/linkedevents/pull/655/commits/bc1d8929666f1700be8f015f76e6b46bb3ae896b